app/views/trainees/show.html.erb

```diff
@@ -24,7 +24,7 @@
   <%= link_to "Update", previous_education_trainee_path(@trainee), class: "govuk-link" %>
 </p>
 
-<%= render_component(SummaryTable::View.new(content_hash: TraineePreviousEducation.new(@trainee).call)) %>
+<%= render_component(SummaryCard::View.new(title: "Previous education", rows:  TraineePreviousEducation.new(@trainee).call)) %>
```

Comment:
a bit random but is this one missing wrapping div like 

```html
<div class="previous-education">
```

Reply:
We did spot this well we were updating but figured this page is likely to be changed drastically.....but if you insist 😉